includes/polling/os/vxoa.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use LibreNMS\RRD\RrdDefinition;
+
+$hardware = snmp_get($device, 'spsProductModel.0', '-Osqv', 'SILVERPEAK-MGMT-MIB');
+$serial = snmp_get($device, 'spsSystemSerial.0', '-Osqv', 'SILVERPEAK-MGMT-MIB');
+$version = snmp_get($device, 'spsSystemVersion.0', '-Osqv', 'SILVERPEAK-MGMT-MIB');
```

Comment:
Hi @shepherdjay 
Could you use snmp_get_multi here instead of 3 snmpget calls ? 
Example in `includes/polling/os/zyxel.inc.php`
Thanx

Reply:
Yep I will take a look and update it